### PR TITLE
chore: add test replay button to debug page

### PR DESCRIPTION
**Pull request by dkasper-was-taken**
### Steps to test
1. Create/link to repository with failing tests
2. Navigate to Debug page
3. Verify Presence of "View Replay" button on test failures

### PR Tasks

- [x] Have tests been added/updated?
- [x] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [n/a] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: packages/app/src/debug/DebugFailedTest.cy.tsx [hunk 1]
```diff
@@ -205,6 +207,7 @@ describe('<DebugFailedTest/>', () => {
       hasStdout: false,
       hasScreenshots: false,
       hasVideo: false,
+      hasReplay: false,
```
Comment (warrensplayer) [inline packages/app/src/debug/DebugFailedTest.cy.tsx:210]: Add a test case for the new button below.

**Review by warrensplayer — APPROVED** [review]
One request for an addition to an existing test.  Otherwise good to go.  Going to conditionally approve assuming you will add that.

**Comment by warrensplayer** [comment]
@dkasper-was-taken I removed my approval after looking at the typescript linting failure with the `check-ts` workflow.  Looks like the icon update from the design systems is causing an issue.

File: packages/app/src/debug/DebugFailedTest.cy.tsx (hunk 1 above)
Comment (dkasper-was-taken) [reply packages/app/src/debug/DebugFailedTest.cy.tsx:210]: Updated.

**Review by AtofStryker — APPROVED** [review]

**Review by warrensplayer — APPROVED** [review]

**Review by warrensplayer — APPROVED** [review]

**Review by ryanthemanuel — APPROVED** [review]

**Pull request merged**